pkg/querier/querier.go

```diff
@@ -72,3 +72,18 @@ func GetSelfNode(isAgent bool, node string) v1.ObjectReference {
 func GetVersion() string {
 	return version.GetFullVersion()
 }
+
+// NetworkPolicyQueryFilter is used to filter the result while retrieve network policy
+// Not settled attributes means match all.
```

Comment:
Not settled attributes? How about empty attributes?

Reply:
Do you mean users want to filter policies with some attributes that are empty? For now, I don't think using an empty attribute to filter policies will make any sense to users.

Reply:
No, I was asking if "not settled attributes" is correct or not. I think you mean "empty attribute mean match all" (if an attribute is empty in the filter, it will not be used as a condition, right?)

Reply:
Yeah that's what I mean. I could change the description to make it more clear.